templates/log.html

```diff
@@ -1,37 +1,91 @@
+
+    <!-- Custom styles for this template -->
+    <link href="http://getbootstrap.com/examples/starter-template/starter-template.css" rel="stylesheet">
+
+    <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
+    <!--[if lt IE 9]><script src="../../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
+    <script src="http://getbootstrap.com/assets/js/ie-emulation-modes-warning.js"></script>
+
+    <!-- Custom styles for this template -->
+    <link href="/static/style.css" rel="stylesheet">
+
   </head>
 
   <body>
-    <table class="log_metadata">
-      <tr>
-        <th>Command:</th>
-        <td>{{ entry.command }}</td>
-      </tr>
-      <tr>
-        <th>Timestamp:</th>
-        <td>{{ entry.timestamp }}</td>
-      </tr>
-      <tr>
-        <th>Return Code:</th>
-        <td>{{ entry.rc }}</td>
-      </tr>
-    </table>
-
-    <table class="log_metadata">
-      {% set sorted_env = entry.env.items() %}
-      <!-- {{ sorted_env.sort() }} -->
-      {% for k, v in sorted_env %}
-      <tr>
-        <th>{{ k }}</th>
-        <td>{{ v }}</td>
-      </tr>
-      {% end %}
-    </table>
 
-    <div class="log_text">{% for line, is_err in entry.scan_text() %}{% if is_err %}<span class="log_error">{{ line.rstrip() }}</span>{% else %}{{ line.rstrip() }}{% end %}
-{% end %}</div>
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+      <div class="container">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="/">Cronitor Job Dashboard</a>
+        </div>
+        <div id="navbar" class="collapse navbar-collapse">
+          <ul class="nav navbar-nav">
+            <!-- <li class="active"><a href="#">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li> -->
+          </ul>
+        </div><!--/.nav-collapse -->
+      </div>
+    </nav>
+
+    <div class="container">
+      <h1>Logs for: {{ entry.timestamp }}</h1>
+      <table class="table">
+        <tr>
+          <th>Command:</th>
+          <td>{{ entry.command }}</td>
+        </tr>
+        <tr>
+          <th>Timestamp:</th>
+          <td>{{ entry.timestamp }}</td>
+        </tr>
+        <tr>
+          <th>Return Code:</th>
+          <td>{{ entry.rc }}</td>
+        </tr>
+      </table>
+
+      <!-- <table class="table">
+        {% set sorted_env = entry.env.items() %} -->
+        <!-- {{ sorted_env.sort() }} -->
+        <!-- {% for k, v in sorted_env %}
+        <tr>
+          <th>{{ k }}</th>
+          <td>{{ v }}</td>
+        </tr>
+        {% end %}
+      </table> -->
```

Comment:
A lot of this information is pretty useful for diagnosing jobs that have failed; not sure if there's an easy way in Bootstrap to hide it behind a pop-down or similar (to reduce clutter), but it probably shouldn't be left out entirely.